src/ecrc-frontend/src/components/base/header/Header.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export default function Header({ name }) {
```

Comment:
let's use the same pattern discussed earlier:
```suggestion
export default function Header({ header: { name } }) {
```

Reply:
Fixed